static/script/devices/mediaplayer/html5.js

```diff
@@ -124,6 +124,7 @@ require.def(
                         this._toBuffering();
                         this._targetSeekTime = this._getClampedTimeForPlayFrom(seconds);
                         if (this._isNearToCurrentTime(this._targetSeekTime)) {
+                            this._targetSeekTime = undefined;
```

Comment:
To clarify, this is the only place where _targetSeekTime 'leaked out' of playFrom, and remained set even though no seek would ever take place. The rest of this change is about performing deferred seeks on finishing buffering, so any time we exited buffering after this (eg actual buffering, seeking etc) an unwanted deferred seek would be performed because _targetSeekTime still had a value in.